site/content/docs/5.2/components/card.md

```diff
@@ -71,7 +73,7 @@ Subtitles are used by adding a `.card-subtitle` to a `<h*>` tag. If the `.card-t
 <div class="card" style="width: 18rem;">
   {{< placeholder width="100%" height="180" class="card-img-top" text="Image cap" >}}
   <div class="card-body">
-    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    <p class="card-text fw-bold">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
```

Comment:
I don't think so since we don't want to force people using the `.fw-bold` since many cards don't have this `.fw-bold` content and people might not want it. Any thoughts @julien-deramond ?